### DISCUSSION TITLE
Add "Turn off device" to list of Quick actions

### DISCUSSION THREAD

**Discussion by speleoluc** [osmandapp/OsmAnd#14932] [Ideas]
When operated through a waterproof case, the tablet's Power button is not always accessible. I can turn on my (Samsung) tablet with a double tap, but to turn it off, I must leave OSMAnd+ and return to the Android desktop. A Power Off quick action would be most convenient.

**Comment by vshcherb** [comment]
I don't think it's possible actually cause we don't have API for that
